### PR TITLE
add sshd service

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: services/sshd.nix [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{ config, lib, pkgs, ... }:
+{
+  services.openssh = {
+    enable = true;
+    passwordAuthentication = false;
```
Comment (Mic92) [inline services/sshd.nix:5]: I believe everyone is using ssh keys?

Comment (adisbladis) [reply services/sshd.nix:5]: Yes, in fact we've always had `services.openssh.passwordAuthentication = false;` set (in `profiles/security.nix`).

Comment (Mic92) [reply services/sshd.nix:5]: Ok. I would still keep it here as well because I am importing this file into the kexec image.

Comment (adisbladis) [reply services/sshd.nix:5]: Sounds good to me! Just wanted to point out that we are already banning password logins :)

**Review by zimbatm — APPROVED** [review]

**Pull request merged**